torchvision/datasets/utils.py

```diff
@@ -95,7 +95,10 @@ def download_url(url, root, filename=None, md5=None):
                 )
             else:
                 raise e
-
+        # check integrity of downloaded file
+        if not check_integrity(fpath, md5):
+            print("Error downloading file: Dataset not found in " + fpath + " or corrupted.")
+            raise RuntimeError("Dataset not found or corrupted.")
```

Comment:
Is there a need to print a message and raise a `RuntimeError`? Can't you just raise the error?